debian/control

```diff
@@ -7,15 +7,59 @@ Build-Depends: debhelper (>=9.0.0)
 
 Package: kano-desktop
 Architecture: all
-Depends: ${misc:Depends}, openbox (>=3.5.2-4~kano.1),
-         kdesk (>=1.4-1), kano-themes, lxmenu-data (>=0.1.2-3~kano), kano-video (>= 1.2-1),
-         kano-video-files (>=1.1-2), kano-init-flow (>=2.2-1), kano-profile (>=2.2.0-4),
-         libkdesk-dev, kano-greeter, kano-updater (>=3.0.0-1), kano-home-button,
-         kano-apps (>=2.0-1), kano-settings (>=2.0-1), kano-content, telnet,
-         kano-widgets (>=2.2-1), kano-applets, chromium-browser (>= 41.0), kano-dashboard(>=0.1),
-         rpi-chromium-mods-kano, chromium
+Depends:
+    ${misc:Depends},
+    openbox (>=3.5.2-4~kano.1),
+    kdesk (>=1.4-1),
+    kano-wallpapers,
+    kano-themes,
+    lxmenu-data (>=0.1.2-3~kano),
+    kano-video (>= 1.2-1),
+    kano-video-files (>=1.1-2),
+    kano-init-flow (>=2.2-1),
+    kano-profile (>=2.2.0-4),
+    libkdesk-dev,
+    kano-greeter,
+    kano-updater (>=3.0.0-1),
+    kano-home-button,
+    kano-apps (>=2.0-1),
+    kano-settings (>=2.0-1),
+    kano-content,
+    telnet,
+    kano-widgets (>=2.2-1),
+    kano-applets,
+    chromium-browser (>= 41.0),
+    kano-dashboard(>=0.1),
+    rpi-chromium-mods-kano,
+    chromium
```

Comment:
Actually this was done without breaking anything in dashboard. The migrated files were moved to a completely new location so if the dashboard wants to use them then great, if not then it is free to go about its business.